src/briefcase/integrations/android_sdk.py

```diff
@@ -63,16 +91,12 @@ def verify_android_sdk(command):
     print("Setting up Android SDK...")
     try:
         sdk_zip_path = command.download_url(
-            url=sdk.sdk_url,
-            download_path=command.dot_briefcase_path / "tools",
+            url=sdk.sdk_url, download_path=command.dot_briefcase_path / "tools",
         )
     except requests_exceptions.ConnectionError:
         raise NetworkFailure("download Android SDK")
     try:
-        command.shutil.unpack_archive(
-            str(sdk_zip_path),
-            extract_dir=str(sdk.root_path)
-        )
+        command.shutil.unpack_archive(str(sdk_zip_path), extract_dir=str(sdk.root_path))
```

Comment:
Pedantic: don't mix your code style pull requests with your pull requests that add features :P 

It's fine; I'm just making fun. :)

Reply:
ppfffffft :-P